search.html

```diff
@@ -91,13 +91,7 @@
         color: #000000;
         font-weight: normal;
         -webkit-box-shadow: 1px 1px 2px rgba(0,0,0,0.3);
-        -moz-box-shadow: 1px 1px 2px rgba(0,0,0,0.3);
-        box-shadow: 1px 1px 2px rgba(0,0,0,0.3);
```

Comment:
box-shadow is the standard form for this property and ought to be kept here. The -webkit and -moz versions are browser-specific extensions.